Share The Load/Model/Question.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Question.swift
+//  Share The Load
+//
+//  Created by Christopher Hefferman on 12/01/2021.
+//
+
+import Foundation
+import RealmSwift
+
+final class Question: Object, Codable {
+    @objc dynamic var id: Int = 0
+    @objc dynamic var trilogy: String = ""
+    @objc dynamic var film: String = ""
+    @objc dynamic var title: String = ""
+    @objc dynamic var correctAnswerId: Int = 0
+    var answers = List<Answer>()
+    
+    var correctAnswer: Answer {
+        answers.first { $0.id == 1 }!
```

Comment:
Probably a good time to come back to this force unwrap. We're saying we want the app to crash if any question doesn't have an answer. I think we should change this to return an optional instead. Then we can easily filter out ones with potentially nil answers when it's accessed